ComposureWatch/src/Components/SearchImg.jsx

```diff
@@ -0,0 +1,6 @@
+const SearchImg = (attribute, list) => {
```

Comment:
In this case, the `SearchImg` function isn't really a component; it is more of a utility function, so I recommend renaming it to be lowercase, e.g., searchImage or getImage, and putting it into a `utils.js` file outside of the Components directory. (If you decide to keep it that is)

Normally the `UpperCase` convention is applied to classes or components.